frontend/src/components/Wheel/index.module.css

```diff
@@ -3,3 +3,84 @@
   grid-template-columns: auto auto;
   width: 50%;
 }
+
+.circle {
+  width: 25em;
+  height: 25em;
+  border: 1px solid black;
+  position: relative;
+  padding: 0;
+  margin: 1em auto;
+  border-radius: 50%;
+  list-style: none;
+  overflow: hidden;
+}
+
+.prize {
+  overflow: hidden;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 50%;
+  height: 50%;
+  transform-origin: 0% 100%;
+}
+
+.text {
+  position: absolute;
+  left: -100%;
+  width: 200%;
+  height: 200%;
+  text-align: center;
+  display: block;
+  padding-top: 20px;
+  transform: skewY(60deg) rotate(15deg);
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.arrow {
+  width: 0;
+  height: 0;
+  border-left: 15px solid transparent;
+  border-right: 15px solid transparent;
+  border-top: 50px solid black;
+  position: absolute;
+  left: 49%;
+  top: 19.5%;
+  z-index: 1;
+}
+
+.spin_button {
+  position: absolute;
+  left: 46%;
+  width: 120px;
+  height: 40px;
+  background-color: blue;
+  border-radius: 30px;
+  margin-top: 20px;
+  cursor: pointer;
+  outline: none;
+  color: white;
+  font-size: 18px;
+}
+
+.start-rotate {
+  animation: start-rotate 1s linear infinite;
+}
+
+@keyframes start-rotate {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.stop-rotate {
+  animation-play-state: paused;
+}
+
+.img_prizes {
+  width: 30px;
+  height: 30px;
+}
```

Comment:
Why do we have two stylesheets for this component? Which one is used?

Reply:
Stylesheet was removed